spec/system/order_spec.rb

```diff
@@ -3,35 +3,34 @@
 require 'rails_helper'
 
 RSpec.describe 'Order check', type: :system do
-  before do
-    driven_by(:rack_test)
-  end
-
   context 'when an employee purchase a reward and check order list' do
     let!(:employee) { create(:employee) }
     let(:admin_user) { create(:admin_user) }
     let(:company_value) { create(:company_value) }
-    let!(:reward) { create(:reward, price: 1) }
-    let(:reward2) { create(:reward, price: 99) }
+    let!(:reward) { create(:reward, price: 2) }
```

Comment:
I would add here info, that this reward has the post delivery method explicitly -  it may be hard to catch later on